src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj

```diff
@@ -12,7 +12,6 @@
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests</AssemblyName>
     <TargetExt>.dll</TargetExt>
     <TargetFramework>netstandard1.3</TargetFramework>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```

Comment:
@jaredpar It should eventually change the icon of the projects in Solution Explorer, but I was never told where to file a bug for that: https://github.com/dotnet/roslyn/pull/18363#issuecomment-299922406

In the meantime, it's used by LUT to filter out projects that can't have tests.